inc/Classes/DB.php

```diff
@@ -110,11 +110,13 @@ public function connect($save = false)
         $user = $config['database']['user'];
         $pass = $config['database']['passwd'];
         $database = $config['database']['database'];
+        $charset = $config['database']['dbport'];
```

Comment:
Is there a default value if this is not set in the config? `3306` would be a good value.

Reply:
It would even be better if not `$charset` would be overwritten, but `$dbport` ;)

Reply:
resolved with new commit